pybossa/cache/site_stats.py

```diff
@@ -66,13 +75,20 @@ def n_anon_users():
 
 @memoize_with_l2_cache(timeout=ONE_DAY, timeout_l2=TWO_WEEKS,
                        key_prefix="site_n_tasks")
-def n_tasks_site():
+def n_tasks_site(days='all'):
     """Return number of tasks in the server."""
-    sql = text('''SELECT COUNT(task.id) AS n_tasks FROM task''')
-    results = session.execute(sql)
-    for row in results:
-        n_tasks = row.n_tasks
-    return n_tasks or 0
+    if days == 'all':
+        sql = text('''SELECT COUNT(task.id) AS n_tasks FROM task;''')
```

Comment:
Minor, if you want to, perhaps you can have a common "select_clause" variable, and then depending on the condition, concatenate with "where_clause".

Reply:
Updated via https://github.com/bloomberg/pybossa/pull/824/commits/8713e9014d38d374f90424e4b236e40a7b9cd9a2